Week3/server.js

```diff
@@ -0,0 +1,155 @@
+'use strict'
+
+const express = require('express');
+const todoMethodList = require('./app');
+const fs = require('fs');
+const bodyParser = require('body-parser');
+
+const app = express();
+app.use(bodyParser());
+
+const port = 3000;
+
+//categories
+app.get('/categories', async (req, res) => {
+  const msg = await todoMethodList.listCategory();
+  res.send(JSON.parse(msg));
+});
+//{"category": { "name": "Sport Lesson" }}
+app.post('/categories', async (req, res) => {
+  const msg = await todoMethodList.createCategory(JSON.parse(JSON.stringify(req.body.category)).name);
```

Comment:
I didnt understand the ```JSON.parse(JSON.stringify(req.body.category))```
seems to me that the result of it will be exactly the original value of ```req.body.category```. So you could simply just use ```req.body.category.name```. Also, try to do some input verification first. That would prevent potential errors. =)

![image](https://user-images.githubusercontent.com/197033/52174383-24a54f00-2793-11e9-935a-91cb7687274b.png)